packages/fxa-settings/src/pages/Signin/SigninTokenCode/container.test.tsx

```diff
@@ -169,36 +156,26 @@ describe('SigninTokenCode container', () => {
         mockLocationState = createMockSigninLocationState();
       });
 
-      it('displays loading spinner when loading', () => {
-        mockTotpStatusQuery.mockImplementation(() => {
-          return {
-            data: null,
-            loading: true,
-          };
+      it('redirects to totp screen if user has totp enabled', async () => {
+        mockHasTotpAuthClient = true;
+        await act(async () => {
```

Comment:
Alternatively, this could be reduced to
```await waitFor(()=>render());```

Also, can't comment at top of file but looks like there's an unused `ApolloModule` import in this file.